go.mod

```diff
@@ -52,7 +52,7 @@ require (
 	k8s.io/klog/v2 v2.8.0
 	k8s.io/kube-aggregator v0.21.1
 	k8s.io/utils v0.0.0-20201110183641-67b214c5f920
-	sigs.k8s.io/kube-storage-version-migrator v0.0.3
+	sigs.k8s.io/kube-storage-version-migrator v0.0.4
```

Comment:
bumped to import `sigs.k8s.io/kube-storage-version-migrator/pkg/clients/clientset/fake` in testing.